src/resolvers/group/index.ts

```diff
@@ -30,11 +39,10 @@ const addGroup: MutationResolvers['addGroup'] = async (
     groupType: input.groupType,
     primaryLocation: input.primaryLocation,
     primaryContact: input.primaryContact,
-    website: input.website
+    website: input.website || undefined,
```

Comment:
My linter was complaining about `input.website` potentially being `string | null | undefined` and said only `string | undefined`, which seems odd because it's nullable column per the migration file, but maybe we have to call that out explicitly in the model like:

```typescript
@Column
public website?: string | null
```

Not really sure. Still figuring out how these sequelize models work.

Reply:
i'm not sure -- i wasn't having issues with that particularly bit! however i am curious about this type:
` extends Optional<GroupAttributes, 'id'> {}` -- most of our fields are not optional, and it seems like (i am in a rush so maybe reading this incorrectly) this picks the key out of the second argument and makes it non-optional, which is probably not ... a thing we want?